spec/base_spec_helper.rb

```diff
@@ -162,6 +162,9 @@
       vcr_config.default_cassette_options = { record: :none } if ENV["CI"]
     end
     example.run
+    VCR.configure do |vcr_config|
+      vcr_config.cassette_library_dir = "spec/fixtures/vcr_cassettes"
+    end
```

Comment:
It seems VCR was looking for a cassette in the Stripe folder, although this one is not related to Stripe:

```
       VCR is currently using the following cassette:
         - /home/runner/work/openfoodnetwork/openfoodnetwork/spec/fixtures/vcr_cassettes/Stripe-v10.3.0/ConnectAppJob/stores_connection_data_on_the_app.yml
```

Somehow, the around block on base spec helper is not really resetting the VCR configs. This change might just do it, but I'm not sure there isn't a better way.

Reply:
This makes sense: we are setting a different config value, and need to re-set it back after the example is completed. This would be relevant for the other config change too (`default_cassette_options`).

I couldn't find a built-in method for achieving this, so would be tempted to build one..
For  now, I'd suggest saving the current vcr_config before `example.run`, then applying it again after.

I've pushed a new commit to demonstrate, let me know what you think.